modules/optimization/examples/bimaterial_mesh/synthetic_data.csv

```diff
@@ -0,0 +1,12 @@
+id,temperature,x,y,z
+0,1.5499999950466,0,-3.99,0
+0.798,122.15999960961,0,-3.192,0
+1.596,230.74999926258,0,-2.394,0
+2.394,326.45999895672,0,-1.596,0
+3.192,409.18999869233,0,-0.798,0
+3.99,479.99999846604,0,0,0
+4.788,508.43499922095,0,0.798,0
+5.586,530.90999981763,0,1.596,0
+6.384,546.895000242,0,2.394,0
+7.182,556.44000049541,0,3.192,0
+7.98,559.97500058926,0,3.99,0
```

Comment:
Would you mind reminding me why we have ids that are not integers? Also, are the data sampled from a forward solve or just randomly created?

Reply:
lineValueSampler creates will create this type of id if the data is sorted_by = y